src/main/java/ratpack/zipkin/internal/ZipkinHttpClientImpl.java

```diff
@@ -105,14 +109,63 @@ public void close() {
         return request(uri, requestConfigurer.prepend(RequestSpec::post));
     }
 
-    private Action<? super RequestSpec> tracedRequestAction(final Action<? super RequestSpec> action) {
-        return action.append(requestSpec -> {
```

Comment:
alternatively we can make `ratpack.zipkin.internal.MethodCapturingRequestSpec` modifies the actual `ratpack.http.client.RequestSpec` in its spec methods